ansible/vars/topo_t0-u254d2.yml

```diff
@@ -0,0 +1,5370 @@
+      vm_offset: 222
+    ARISTA224T1:
+      vlans:
+        - 225
+      vm_offset: 223
+    ARISTA225T1:
+      vlans:
+        - 226
+      vm_offset: 224
+    ARISTA226T1:
+      vlans:
+        - 227
+      vm_offset: 225
+    ARISTA227T1:
+      vlans:
+        - 228
+      vm_offset: 226
+    ARISTA228T1:
+      vlans:
+        - 229
+      vm_offset: 227
+    ARISTA229T1:
+      vlans:
+        - 230
+      vm_offset: 228
+    ARISTA230T1:
+      vlans:
+        - 231
+      vm_offset: 229
+    ARISTA231T1:
+      vlans:
+        - 232
+      vm_offset: 230
+    ARISTA232T1:
+      vlans:
+        - 233
+      vm_offset: 231
+    ARISTA233T1:
+      vlans:
+        - 234
+      vm_offset: 232
+    ARISTA234T1:
+      vlans:
+        - 235
+      vm_offset: 233
+    ARISTA235T1:
+      vlans:
+        - 236
+      vm_offset: 234
+    ARISTA236T1:
+      vlans:
+        - 237
+      vm_offset: 235
+    ARISTA237T1:
+      vlans:
+        - 238
+      vm_offset: 236
+    ARISTA238T1:
+      vlans:
+        - 239
+      vm_offset: 237
+    ARISTA239T1:
+      vlans:
+        - 240
+      vm_offset: 238
+    ARISTA240T1:
+      vlans:
+        - 241
+      vm_offset: 239
+    ARISTA241T1:
+      vlans:
+        - 242
+      vm_offset: 240
+    ARISTA242T1:
+      vlans:
+        - 243
+      vm_offset: 241
+    ARISTA243T1:
+      vlans:
+        - 244
+      vm_offset: 242
+    ARISTA244T1:
+      vlans:
+        - 245
+      vm_offset: 243
+    ARISTA245T1:
+      vlans:
+        - 246
+      vm_offset: 244
+    ARISTA246T1:
+      vlans:
+        - 247
+      vm_offset: 245
+    ARISTA247T1:
+      vlans:
+        - 248
+      vm_offset: 246
+    ARISTA248T1:
+      vlans:
+        - 249
+      vm_offset: 247
+    ARISTA249T1:
+      vlans:
+        - 250
+      vm_offset: 248
+    ARISTA250T1:
+      vlans:
+        - 251
+      vm_offset: 249
+    ARISTA251T1:
+      vlans:
+        - 252
+      vm_offset: 250
+    ARISTA252T1:
+      vlans:
+        - 253
+      vm_offset: 251
+    ARISTA253T1:
+      vlans:
+        - 254
+      vm_offset: 252
+    ARISTA254T1:
+      vlans:
+        - 255
+      vm_offset: 253
+  DUT:
+    vlan_configs:
+        default_vlan_config: one_vlan_per_intf
+        one_vlan_per_intf:
+        Vlan1000:
+            id: 1000
+            intfs: [0]
+            prefix_v6:  fc00:c:c:0001::/64
+            tag: 1000
+        Vlan1001:
+            id: 1001
+            intfs: [1]
+            prefix_v6:  fc00:c:c:0002::/64
+            tag: 1001
+
+configuration_properties:
+  common:
+    dut_asn: 4200000000
+    dut_type: ToRRouter
+    swrole: leaf
+    podset_number: 200
+    tor_number: 16
+    tor_subnet_number: 2
+    max_tor_subnet_number: 16
+    tor_subnet_size: 128
+    spine_asn: 4200200000
+    leaf_asn_start: 4200100000
+    tor_asn_start: 4200000000
+    failure_rate: 0
+    nhipv6: FC0A::FF
+
+configuration:
+  ARISTA01T1:
+    properties:
+    - common
+    bgp:
+      router-id: 0.12.0.3
```

Comment:
may you double check the router-id with Jing? Not sure if router-id can start with 0.x.x.x range.

Reply:
0.x.x.x works,
I tested it in lab, for example there is a bgp session between SONiCM0 and ARISTA01MX, I assigned router-id 0.12.0.0 to SONiCM0 and 0.12.0.1 to ARISTA01MX, after change, bgp session is restarted and established. 

Below are ipv4 bgp summary on m0 and mx, we can see router-id is 0.12.0.x and bgp session are established.

```
bjw2-can-7215a1-1# show ip bgp sum

IPv4 Unicast Summary (VRF default):
BGP router identifier 0.12.0.0, local AS number 65100 vrf-id 0
.....

Neighbor        V         AS   MsgRcvd   MsgSent   TblVer  InQ OutQ  Up/Down State/PfxRcd   PfxSnt Desc
....
10.0.0.65       4      64001     32061     32063        0    0    0 00:00:12            1        4 ARISTA01MX
...



ARISTA01MX>show ip bgp sum
BGP summary information for VRF default
Router identifier 0.12.0.1, local AS number 64001
Neighbor Status Codes: m - Under maintenance
  Description              Neighbor         V  AS           MsgRcvd   MsgSent  InQ OutQ  Up/Down State   PfxRcd PfxAcc
  65100                    10.0.0.64        4  65100             66        65    0    0 00:00:14 Estab   3      3
```

Reply:
as long as the router-id is not zero, it should work.  If we can use something similar with real deployment, it will be better.  but I'm okay with this too.